src/components/MovieShowPage/MovieShowPage.js

```diff
@@ -42,6 +44,9 @@ const MovieShowPage = (props) => {
     <article className='show-page'>
       <img className='backdrop-img' src={movie.backdrop_path} />
       <section className='movie-info'>
+        <Link to='/'>
+          <button className='back-button' type='button'>⬅</button>
```

Comment:
I'm glad you added a back button here, it was something I wanted to add on to our to-do list